ocserv-install.sh

```diff
@@ -4,8 +4,11 @@ install() {
 
 apt update -y
 
-ip=$(hostname -I|cut -f1 -d ' ')
-echo "Your Server IP address is:$ip"
+# need to fill up your domain name or IP here
+# If you want to use domain name, you gotta setup DNS A record at first.
+# For instance: vpn.yourdomain.com x.x.x.x
+ip=''
```

Comment:
instead of making it '', let's make it optional so if user skipped and pushed enter, it uses the IP otherwise first ask user to add their domain or anything they want